app/views.py

```diff
@@ -438,20 +539,39 @@ def quiz():
         submit_response(id, request.form)
         return redirect(url_for('quiz'))
 
-@app.route('/result')
+@app.route('/quiz/<int:quizID>/<int:qnNum>', methods=['GET','POST'])
```

Comment:
was rerouted from educator's dashboard, hence not checking for validation? But probably should still just in case someone can figure out the quiz id and qn num

Reply:
Oh this is not educator's preview, this is student taking educator's quiz. Will validate student.